core/modules/qdisp/QueryRequest.cc

```diff
@@ -153,6 +154,13 @@ bool QueryRequest::_importStream() {
     bool retrieveInitiated = false;
     // Pass ResponseRequester's buffer directly.
     std::vector<char>& buffer = _requester->nextBuffer();
+    LOGF_DEBUG("QueryRequest::_importStream buffer.size=%1%" % buffer.size());
+    for(auto iter=buffer.begin(); iter != buffer.end(); iter++) *iter=8;
+    std::ostringstream dbg;
+    const void* pbuf = (void*)(&buffer[0]);
+    dbg << "_importStream->GetResponseData size=" << buffer.size();
+    dbg << ' ' << pbuf << ' ' <<  util::prettyCharList(buffer, 25);
+    LOGF_INFO(dbg.str().c_str());
```

Comment:
YOu could probably do all of that without temporary ostringstream:

    LOGF_INFO("_importStream->GetResponseData size=%1% %2% %3%" % 
              buffer.size() % pbuf % util::prettyCharList(buffer, 25));

Otherwise do `LOGF_INFO("%1%" % dbg.str());` as it may have % in the resulting string